editors/vscode/client/extension.ts

```diff
@@ -70,7 +69,8 @@ export async function activate(context: ExtensionContext) {
   const traceOutputChannel = window.createOutputChannel(traceOutputChannelName);
 
   const ext = process.platform === "win32" ? ".exe" : "";
-  const command = join(context.extensionPath, `./target/release/oxc_vscode${ext}`);
+  let serverPath = process.env.SERVER_PATH_DEV ?? `oxc_vscode${ext}`;
```

Comment:
https://github.com/oxc-project/oxc/blob/282b79d05d2919b54cf94b7c060658e801112e23/.vscode/launch.json#L14-L14, the environment defined here, you can change `debug` to `release` if you want to test the language server in production environment.But it will cost a lot of time, usually a debug binary is fairly enough.